src/components/search/searchComponents/Button.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from "react";
+
+class Button extends Component {
+    
+    render() {
+        return (
+            <button className="btn btn-primary fn-submit-name" onClick={this.sayHello}>{this.props.name}</button>
```

Comment:
I don't think it's being used anywhere but your `onClick` won't do anything because `this.sayHello` isn't defined